js-miniapp-sdk/test/miniapp.spec.ts

```diff
@@ -43,12 +43,21 @@ describe('getUniqueId', () => {
 });
 
 describe('requestPermission', () => {
+  window.MiniAppBridge.requestCustomPermissions.resolves({
+    permissions: [
+      {
+        name: CustomPermissionName.LOCATION,
+        status: CustomPermissionStatus.ALLOWED,
+      },
+    ],
+  });
+
   it('should delegate to requestPermission function when request any permission', () => {
     const spy = sinon.spy(miniApp, 'requestPermission' as any);
 
     miniApp.requestLocationPermission();
 
-    return expect(spy.callCount).to.equal(1);
+    return expect(spy.callCount).to.equal(0);
```

Comment:
You need to make this test case work properly 😆

Reply:
You can use the expectation with a promise like this:
```
return miniApp.requestLocationPermission()
  .then(() -> expect(...))
```

Reply:
Thanks, I have fixed this test.